.gitignore

```diff
@@ -0,0 +1 @@
+
```

Comment:
- [ ] There is a .gitignore file listing `test.md` file and `node_modules` directory.